hyperspy/gui/egerton_quantification.py

```diff
@@ -369,10 +369,12 @@ def _interpolator_kind_changed(self, old, new):
         self.span_selector_changed()
 
     def _ss_left_value_changed(self, old, new):
-        self.span_selector_changed()
+        if not np.isnan(self.ss_right_value):
+            self.span_selector_changed()
 
     def _ss_right_value_changed(self, old, new):
-        self.span_selector_changed()
+        if not np.isnan(self.ss_right_value):
```

Comment:
It should be fine now, thanks.
On Wed, 2015-07-29 at 13:53 -0700, vidartf wrote:
> In hyperspy/gui/egerton_quantification.py:
> >  
> >      def _ss_right_value_changed(self, old, new):
> > -        self.span_selector_changed()
> > +        if not np.isnan(self.ss_right_value):
> Did you want to push those tests here, or should I merge as it is?
> —
> Reply to this email directly or view it on GitHub.